### ISSUE TITLE
Version mismatch between vue and vue-server-renderer

### ISSUE THREAD

**Issue by ThomRick** [AngelMunoz/nesting-nuxt#2]
After npm install and npm start the following trace appears:
```
 thomrick  ⎇  buefy S:1 ?:1  ~/Documents/OpenSourceProjects/nesting-nuxt   npm start

> nest-typescript-starter@1.0.0 start /Users/thomrick/Documents/OpenSourceProjects/nesting-nuxt
> node index.js


/Users/thomrick/Documents/OpenSourceProjects/nesting-nuxt/node_modules/vue-server-renderer/index.js:8
  throw new Error(
        ^
Error:

Vue packages version mismatch:

- vue@2.4.1
- vue-server-renderer@2.3.4

This may cause things to work incorrectly. Make sure to use the same version for both.

    at Object.<anonymous> (/Users/thomrick/Documents/OpenSourceProjects/nesting-nuxt/node_modules/vue-server-renderer/index.js:8:9)
    at Module._compile (module.js:569:30)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/thomrick/Documents/OpenSourceProjects/nesting-nuxt/node_modules/ts-node/src/index.ts:384:14)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/thomrick/Documents/OpenSourceProjects/nesting-nuxt/node_modules/nuxt/dist/webpack:/external "vue-server-renderer":1:1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nest-typescript-starter@1.0.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the nest-typescript-starter@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
I use node 8 and npm 5

When I fixed the vue (by adding it in the package.json) version to be the same as vue-server-render everything is ok.

**Comment by AngelMunoz** [comment]
hey thanks for the feedback yes forgot to pin versions, do you think it would be nice to add a small controller of sample data just to showcase?  I think I'll have some free time on the weekend and may do something along that also any other ideas that may come up?

**Comment by AngelMunoz** [comment]
closed in efdb83a6e8a886a9053548260dc380a9089b89fb
